pkg/querier/querier.go

```diff
@@ -119,28 +121,31 @@ func New(cfg Config, distributor Distributor, chunkStore ChunkStore) (storage.Qu
 }
 
 // NewQueryable creates a new Queryable for cortex.
-func NewQueryable(dq, cq storage.Queryable, distributor Distributor, cfg Config) storage.Queryable {
+func NewQueryable(distribQuerier, storeQuerier storage.Queryable, chunkIterFn chunkIteratorFunc, cfg Config) storage.Queryable {
 	return storage.QueryableFunc(func(ctx context.Context, mint, maxt int64) (storage.Querier, error) {
 		q := querier{
-			distributor: distributor,
 			ctx:         ctx,
```

Comment:
Is `ctx` still used?

Reply:
Good catch. It actually isn't, but I think it would make sense to use it in Select while waiting for individual results from queriers.